frontend/src/app/main/streams.cljs

```diff
@@ -114,7 +113,7 @@
                   (->> st/stream
                        (rx/filter keyboard-event?)
                        (rx/filter #(let [key (:key %)]
-                                     (= key KeyCodes.ALT)))
+                                     (= key "Alt")))
```

Comment:
These i think could be added to keyboard as a new function like the others

Reply:
done